.env

```
@@ -0,0 +1,25 @@
+# Configurações do Servidor
+
+SERVER_PORT=8080
+
+# Configurações de Banco de Dados
+
+DATASOURCE_URL=jdbc:jdbc:postgresql://ep-dawn-butterfly-ac9xwagt-pooler.sa-east-1.aws.neon.tech/neondb?&sslmode=require&channelBinding=require
+DATASOURCE_USERNAME=neondb_owner
+DATASOURCE_PASSWORD=npg_tpCjYfF2inu4
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATASOURCE_PASSWORD key should go before the DATASOURCE_URL key

(UnorderedKey)